modules/index/src/main/java/org/apache/ignite/internal/index/ChangeIndexStatusTask.java

```diff
@@ -333,8 +335,26 @@ private <T> CompletableFuture<T> inBusyLocks(Supplier<CompletableFuture<T>> supp
 
         try {
             return supplier.get();
+        } catch (Throwable t) {
+            return failedFuture(t);
         } finally {
             leaveBusy();
         }
     }
+
+    private MetaIndexStatusChange statusChange() {
+        IndexMeta indexMeta = indexMetaStorage.indexMeta(indexDescriptor.id());
+
+        if (indexMeta == null) {
+            // Index was destroyed under a low watermark, well, we need to build it.
```

Comment:
The comment says that the index was already destroyed, but it also says that it needs to build it. Is this a typo? Should it say that it does NOT need to build it anymore?

Reply:
I made a mistake, I fixed it.